src/validate-text.ts

```diff
@@ -115,15 +115,15 @@ async function validateText(textToBeValidated: string, options?: ValidateTextOpt
 	cssValidationResponse.errors?.forEach((error) => {
 		result.errors.push({
 			line: error.line,
-			message: error.message,
+			message: error.message.replace(' : ', '').trim(),
```

Comment:
I worry this is too broad. We could accidentally remove colons and spaces inside the message itself. Instead, let's use this RegEx, which removes all trailing spaces/colons:

```suggestion
			message: error.message.replace(/[ :]+$/, '').trim(),
```